compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp

```diff
@@ -464,6 +465,52 @@ FailureOr<Flow::DispatchRegionOp> Flow::wrapOpInDispatchRegion(
   return newRegionOp;
 }
 
+//===---------------------------------------------------------------------===//
+// UnCSE Clonable ops
+//===---------------------------------------------------------------------===//
+
+namespace {
+struct UnCseClonableOps : public RewritePattern {
+  UnCseClonableOps(MLIRContext *context)
+      : RewritePattern(MatchAnyOpTypeTag(), /*benefit=*/1, context) {}
+
+  LogicalResult matchAndRewrite(Operation *op,
+                                PatternRewriter &rewriter) const override {
+    if (!isa<FunctionOpInterface>(op->getParentOp())) {
+      return rewriter.notifyMatchFailure(op, "parent is not a func.func");
+    }
+    if (!Flow::isClonableIntoDispatchOp(op)) {
```

Comment:
style nit for consistency/avoiding potential collisions
```suggestion
    if (!IREE::Flow::isClonableIntoDispatchOp(op)) {
```

noticing that other things that were written for this file are also not doing it so maybe it's a TBD cleanup - going forward would be good to stick with the fully namespaced variant

Reply:
Removed all of this